modules/controllers/pool-controller.ts

```diff
@@ -133,6 +134,7 @@ export function PoolController(tracer?: any) {
                 chain,
                 pools.map(({ id }) => id),
             );
+            await updateLifetimeValues(pools.map(({ id }) => id));
```

Comment:
should we make this its own cron? doesnt need to run as often as updating the liquidity values. Not sure how heavy the query is.

Reply:
Actually, `updateLiquidityValuesForActivePools` is used by v2 and v3. v2 already has another cron `update-lifetime-values-for-all-pools` to update these. So for v2 it would now run twice.